plugin.video.hardstyletv/addon.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.hardstyletv"
+	name="HardstyleTV"
+	version="1.0.0"
+	provider-name="alpadev">
+  <requires>
+    <import addon="xbmc.python" version="2.14.0"/>
```

Comment:
please set xbmc.python version to `2.25.0` to match krypton. See: http://kodi.wiki/view/addon.xml